site.yml

```
@@ -21,460 +21,460 @@ Documents:
         Link: https://raw.githubusercontent.com/minio/minio-py/master/docs/API.md
         Slug: python-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Python Client API Reference
+        SEO Description: The MinIO Python SDK provides detailed code examples for the Java API
+        SEO Keywords: Python SDK, Python Client API
 
       - Golang Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-go/master/README.md
         Slug: golang-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Golang Client Quickstart Guide
+        SEO Description: MinIO | Golang Client Quickstart Guide	The MinIO Go Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: Golang, Go
 
       - Golang Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-go/master/docs/API.md
         Slug: golang-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Golang Client API Reference
+        SEO Description: The MinIO Go Client API reference provides detailed code examples for the Go API
+        SEO Keywords: Golang, Go
 
       - .NET Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-dotnet/master/README.md
         Slug: dotnet-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | .NET Client Quickstart Guide
+        SEO Description: The MinIO .NET Client SDK provides simple APIs to access any Amazon S3 compatible object storage server.
+        SEO Keywords: dotnet, .NET
 
       - .NET Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-dotnet/master/Docs/API.md
         Slug: dotnet-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | .NET Client API Reference
+        SEO Description: The MinIO .NET Client SDK provides detailed code examples for the .NET
+        SEO Keywords: dotnet, .NET
 
   - Cookbook:
       - S3cmd with Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/s3cmd-with-minio.md
         Slug: s3cmd-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | S3cmd with MinIO - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to configure and use S3cmd to manage data with MinIO Server
+        SEO Keywords: S3cmd
 
       - AWS CLI with Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-cli-with-minio.md
         Slug: aws-cli-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | AWS CLI with MinIO- Cookbook/Recipe
```

Can we add a space between MinIO and the slash as follows?

```suggestion
        SEO Title: MinIO | AWS CLI with MinIO - Cookbook/Recipe
```